lib/cocoapods/sandbox/path_list.rb

```diff
@@ -111,7 +105,7 @@ def glob(patterns, options = {})
       #
       # @return [Array<Pathname>]
       #
-      def relative_glob(patterns, options = {})
+      def glob(patterns, options = {})
```

Comment:
Because symlinks could point to somewhere that is not relative to the Podspec, the notion of relative globbing sort of had to be thrown out. Glob will now always return absolute paths